sites/main-site/src/content/events/2025/hackathon-barcelona.mdx

```diff
@@ -77,8 +77,9 @@ See [Helpful resources](#helpful-resources) below for more information.
 
 # Schedule
 
-The hackathon will run from **Tuesday 28th October** to **Wednesday 29th October**.
-More information, like a more detailed schedule will be published here soon.
+The hackathon will run from **Tuesday 28th October** 9AM to **Wednesday 29th October** 5PM.
```

Comment:
- What you had in mind for here, @ewels? 100% of the agenda from the summit website here or a more shortened version like I did?

- @mashehu when do you plan to run the quiz?

Reply:
hmm, usually i do on the second day in the afternoon, but people might be too busy then. how about wednesday morning then, to wake everybody up a bit?